databind/src/main/java/gov/nist/secauto/metaschema/databind/io/xml/MetaschemaXmlReader.java

```diff
@@ -146,8 +146,9 @@ public <CLASS> CLASS read(@NonNull IBoundDefinitionModelComplex definition) thro
       ItemReadHandler handler = new ItemReadHandler(ObjectUtils.notNull(event.asStartElement()));
       Object value = definition.readItem(null, handler);
       if (value == null) {
+        event = reader.peek();
         throw new IOException(String.format("Unable to read data%s",
```

Comment:
Minor, non-blocking: not material to the commit, but do we want to space between the word data and formatting string?

```suggestion
        throw new IOException(String.format("Unable to read data %s",
```

Reply:
The message that gets inserted has the space.
```suggestion
        throw new IOException(String.format("Unable to read data.%s",
```